WeatherApp/ConsoleApp/Commands/ICommand.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace ConsoleApp.Commands;
+
+public interface ICommand
+{
+    public Task Execute();
```

Comment:
"public" is not needed on interface methods

Reply:
Removed "public" access modifier from interface methods.